contrib/static-opus/Makefile

```diff
@@ -0,0 +1,182 @@
+##
+#  Fetch the latest dependencies from upstream
+#  ===========================================
+OGG_ARCHIVE = ogg-master.tar.gz
+OGG_URL = https://github.com/xiph/ogg/archive/master.tar.gz
+
+OPUS_ARCHIVE = opus-master.tar.gz
+OPUS_URL = https://github.com/xiph/opus/archive/master.tar.gz
+
+OPUSFILE_ARCHIVE = opusfile-master.tar.gz
+OPUSFILE_URL = https://github.com/xiph/opusfile/archive/master.tar.gz
+
+# We use speex's pkg.m4 to work around a bug in opusfile's autotools config
+# that still depends on pkgconfig even if DEPS_ env flags are present.
+# Mingw 1.0 (Windows) does not provide pkgconfig, so we need to avoid it.
+OPUSFILE_PKG_URL = https://raw.githubusercontent.com/xiph/speex/master/m4/pkg.m4
+
+##
+#  The audio-codecs rely on accurate floating-point calculations
+#  and will fail to build (Opus) if they detect too agressive optimizations
+#  such as Ofast or ffast-math; therefore we strip just these flags
+#  and step-down -Ofast to -O3 as an acceptable fallback.
+#
+CFLAGS:= $(subst -Ofast,-O3,$(CFLAGS))
+CFLAGS:= $(filter-out -ffast-math,$(CFLAGS))
+CFLAGS:= $(filter-out -funsafe-math-optimizations,$(CFLAGS))
+
+CXXFLAGS:= $(subst -Ofast,-O3,$(CXXFLAGS))
+CXXFLAGS:= $(filter-out -ffast-math,$(CXXFLAGS))
+CXXFLAGS:= $(filter-out -funsafe-math-optimizations,$(CXXFLAGS))
+
+LDFLAGS:= $(subst -Ofast,-O3,$(LDLAGS))
+LDFLAGS:= $(filter-out -ffast-math,$(LDFLAGS))
+LDFLAGS:= $(filter-out -funsafe-math-optimizations,$(LDFLAGS))
+
+##
+#  Common commands and arguments
+#  =============================
+THREADS = $(shell nproc || echo 4)
+CURL = curl -s -L
+EXTRACT = tar --strip 1 -zxof
+
+##
+#  Everything-targets
+#  ==================
+all: ogg opus opusfile
+clean: ogg/clean opus/clean opusfile/clean
+dist-clean:
+	rm -rf ogg opus opusfile include lib share pkg.m4 *.gz
```

Comment:
`all`, `clean`, and `dist-clean` should be listed in `.PHONY` as well.

Reply:
Done